tests/tls/test_explicit_options.py

```diff
@@ -37,12 +37,18 @@ def _test():
             if get_driver_name() in ["javascript"]:
                 self.assertIs("encryption", exc.exception.msg.lower())
                 self.assertIs("trust", exc.exception.msg.lower())
+            elif get_driver_name() in ["java"]:
+                self.assertIn("encryption", exc.exception.msg.lower())
+                self.assertIn("trust", exc.exception.msg.lower())
             else:
                 self.fail("Add expected error type for driver.")
 
         self._server = TlsServer("trustedRoot_thehost")
         for scheme in ("neo4j+s", "neo4j+ssc", "bolt+s", "bolt+ssc"):
             for encrypted in (True, False):
-                for certs in ("None", [], ["customRoot.crt"]):
+                cert_options = [[], ["customRoot.crt"]]
+                if encrypted:
+                    cert_options.append("None")
+                for certs in cert_options:
```

Comment:
I feel like this could be a good candidate for another implementation detail feature flag. Maybe something along the lines of `DETAIL_DEFAULT_CONFIG_INDISTINGUISHABLE = "Detail:DefaultConfigIndistinguishable"`. I have the suspicion this will affect other tests as well.